packages/block-editor/src/components/block-edit/index.js

```diff
@@ -24,6 +29,9 @@ import { store as blockEditorStore } from '../../store';
  */
 export { useBlockEditContext };
 
+// Please do not export this at the package level until we have a stable API.
+export const blockControlsFilterName = uuid();
```

Comment:
Yes, that might work as a first iteration 👍🏻

Reply:
Look like this approach creates flakiness in tests. Not sure why.

Reply:
Ah, figured out why. Hook names must start with a letter and `uuid()` will randomly also generate ids starting with a number.